TestFoundation/TestNSURLSession.swift

```diff
@@ -41,6 +41,10 @@ class TestURLSession : XCTestCase {
 	    ("test_customProtocolResponseWithDelegate", test_customProtocolResponseWithDelegate),
             ("test_httpRedirection", test_httpRedirection),
             ("test_httpRedirectionTimeout", test_httpRedirectionTimeout),
+            ("test_illegalHTTPServerResponses", test_illegalHTTPServerResponses),
```

Comment:
If we're keeping the order of the tests the same as in the test list, this line should be moved to the end.